custom/ewsghana/handlers/__init__.py

```diff
@@ -12,3 +12,5 @@
 
 MS_RESOLVED_STOCKOUTS = _('The following commodities %(products_names)s are now available at the %(ms_type)s. '
                           'Please place your order now.')
+
+STOP_MESSAGE = _('You have requested to stop reminders to this number.')
```

Comment:
Is this the message that was used in the original system?  I think it might be better if we made it more explicit saying something like "You will no longer receive reminders on this phone number", but if that's what the original system had, then it's ok.

Also, is there a opt back in feature? If not then we're probably fine.  But if there is, then we should also say something like "To opt back in, reply with ..."